src/rhelocator/config.py

```diff
@@ -44,3 +44,6 @@
         "RHEL": {"7lvm-gen2": "latest", "8-lvm-gen2": "latest", "9-lvm-gen2": "latest"}
     }
 }
+
+# GCP
```

Comment:
Hi @major :) 
I noticed you created some ascii art headers for AWS and Azure. How can I do the same with GCP? :D 
I only found this [web page](https://patorjk.com/software/taag/#p=display&f=Blocks&t=GCP), but I am not sure which font I have to use?

Reply:
Hah, I used `figlet` for that. 😉

Reply:
Ah :smile: , thanks! Worked :)